### PR TITLE
Replace env var with args

### LINKED ISSUE #52: When passing variables to trigger Prow jobs use arguments instead of env variables.
passing values using env variables works well only when within the same context. When you want to pass data from one application to another it is best to use explicit argument passed when starting the command.

Otherwise we need to keep track of env variable names and it is very easy to brake.

### PR DESCRIPTION
**Pull request by sipian**
Fixes #52

### CODE REVIEW

File: components/prombench/manifests/benchmark/2_loadgen.yaml [hunk 1]
```diff
@@ -73,13 +73,11 @@ spec:
       serviceAccountName: loadgen-scaler
       containers:
       - name: prom-load-generator
-        image: gcr.io/prometheus-test-204522/prometheus-bench-loadgen:v0.1.0
+        image: docker.io/sipian/prombench-loadgen:v2.0.0
```
Comment (krasi-georgiev) [inline components/prombench/manifests/benchmark/2_loadgen.yaml]: seems the changes in this file shouldn't belong in this PR

Comment (sipian) [reply components/prombench/manifests/benchmark/2_loadgen.yaml]: Since I have also replaced env var with args in loadgen, I think this is valid.

Comment (sipian) [reply components/prombench/manifests/benchmark/2_loadgen.yaml]: I have replaced `gcr.io/prometheus-test-204522/prometheus-bench-loadgen` with `docker.io/sipian/prombench-loadgen` because `gcr.io/prometheus-test-204522/prometheus-bench-loadgen` might become inactive after the free trial ends

**Comment by krasi-georgiev** [comment]
Merging so we can continue with the other changes and avoid conflicts.
